syft/codes.py

```diff
@@ -8,6 +8,7 @@ class MSGTYPE(object):
     GET_SHAPE = 7
     SEARCH = 8
     FORCE_OBJ_DEL = 9
+    PROMISE = 10
```

Comment:
is this within the scope of this pull request?

Reply:
removed